### PR TITLE
ADD Textarea

### PR DESCRIPTION
**Pull request by pierry01**
# https://github.com/rbui-labs/rbui-web/pull/118

### CODE REVIEW

**Comment by iseth** [comment]
this is great! lets get the associated component in the docs repo to merge this in.
https://github.com/rbui-labs/PhlexUI-web/

File: lib/rbui/textarea/textarea.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module RBUI
+  class Textarea < Base
+    def initialize(rows: 4, **attrs)
```
Comment (pierry01) [inline lib/rbui/textarea/textarea.rb:5]: I've changed to 4 because shadcn uses it

Comment (pierry01) [reply lib/rbui/textarea/textarea.rb:5]: image

**Comment by pierry01** [comment]
> lets get the associated component in the docs repo to merge this

DONE!

**Pull request merged**